src/mmw/mmw/settings/gwlfe_settings.py

```diff
@@ -463,6 +463,42 @@
     'SSLDR': 4.4,
     'SSLDM': 2.2,
     'WeatherNull': -99999,  # This value is used to indicate NULL in ms_weather dataset
+    'ETGrowCoeff': [
```

Comment:
Sorry, that's for P Factor.

For ET take a look at Class1.vb:4989-4995 `TransMonthData`, and see where it is called from (sorry don't have the file with me right now)